cmd/minikube/cmd/config/profile_list.go

```diff
@@ -154,7 +154,7 @@ func profileStatus(p *config.Profile, api libmachine.API) string {
 
```

Comment:
Thank you for this PR, I think for sake of more clarity what active minikube profile is, in addition to "ActiveKubeContext" Column
we should have "Active miniikube" column,
by the way one idea for a follow up PR, 
we could Later get rid of some of the column by default such as "Port" or K8S version, and make it available only in a --detailed or --long format

Reply:
Hi @medyagh, I have updated this PR with my above comment. 
```
minikube profile list -o table
|-----------|-----------|---------|--------------|------|---------|---------|-------|----------------|--------------------|
|  Profile  | VM Driver | Runtime |      IP      | Port | Version | Status  | Nodes | Active Profile | Active Kubecontext |
|-----------|-----------|---------|--------------|------|---------|---------|-------|----------------|--------------------|
| minikube  | docker    | docker  | 192.168.49.2 | 8443 | v1.28.4 | Running |     1 | *              |                    |
| minikube2 | docker    | docker  | 192.168.58.2 | 8443 | v1.28.4 | Running |     1 |                | *                  |
|-----------|-----------|---------|--------------|------|---------|---------|-------|----------------|--------------------|
```

The JSON output is the same as before due to compatibility reasons. I thought about adding `ActiveProfile`, which has the same data as `Active` with a deprecation note. The deprecation note shall point out, that `Active` will be removed in a future release and `ActiveProfile` shall be used instead.
```
minikube profile list -o json | jq | grep -i active
      "Active": true,
      "ActiveKubeContext": false
      "Active": false,
      "ActiveKubeContext": true
```
Please let me know, what you think.